packages/minicaml/minicaml.0.3/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+version: "0.3"
+maintainer: "sudo-woodo3@protonmail.com"
+authors: ["Alessandro Cheli"]
+homepage: "https://github.com/0x0f0f0f/minicaml"
+bug-reports: "https://github.com/0x0f0f0f/minicaml/issues"
+dev-repo: "git+https://github.com/0x0f0f0f/minicaml.git"
+license: "MIT"
+synopsis: "A simple, didactical, purely functional programming language"
+description: "A simple, didactical, purely functional programming language written for the programming 2 course at the University of Pisa, extended with a simple parser made with Menhir and ocamllex"
+doc: "https://github.com/0x0f0f0f/minicaml"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {dev}
```

Comment:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```
Test should run only when `with-test` is set (triggered by `opam install --with-test`)